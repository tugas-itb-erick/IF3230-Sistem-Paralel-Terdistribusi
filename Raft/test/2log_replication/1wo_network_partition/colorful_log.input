5
set_timeout 1 0
simulate 10
client_send_request 1 + 1
client_send_request 1 + 1
simulate 10
get_state 1
crash 1
set_timeout 2 0
simulate 10
client_send_request 2 - 2
client_send_request 2 - 2
simulate 10
get_state 2
crash 2
set_timeout 3 0
simulate 10
client_send_request 3 - 2
client_send_request 3 - 2
simulate 10
get_state 4
end_simulation