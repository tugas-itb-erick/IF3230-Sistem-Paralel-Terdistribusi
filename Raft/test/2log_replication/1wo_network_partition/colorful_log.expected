S1 leader term:2 voted_for:1 commit_index:2 data:2 logs:T0[0] T2[+1+1]
S2 leader term:3 voted_for:2 commit_index:4 data:-2 logs:T0[0] T2[+1+1] T3[-2-2]
S4 follower term:4 voted_for:3 commit_index:6 data:-6 logs:T0[0] T2[+1+1] T3[-2-2] T4[-2-2]
